back/Dockerfile

```diff
@@ -23,4 +23,5 @@ ENV NODE_ENV=production
 RUN yarn install --production
 
 USER node
+EXPOSE 8080
```

Comment:
Hmm refresh my memory plz: These `EXPOSE` directives are purely for documentation, right?

Reply:
"The EXPOSE instruction does not actually publish the port. It functions as a type of documentation between the person who builds the image and the person who runs the container, about which ports are intended to be published. To actually publish the port when running the container, use the -p flag on docker run to publish and map one or more ports, or the -P flag to publish all exposed ports and map them to high-order ports."
looks like it